ras_backstage/resources/collection_exercise/single_collection_exercise.py

```diff
@@ -24,15 +25,24 @@ def get(short_name, period):
         survey['shortName'] = format_short_name(survey['shortName'])
 
         exercises = collection_exercise_controller.get_collection_exercises_by_survey(survey['id'])
+
         # Find the collection exercise for the given period
         exercise = get_collection_exercise_by_period(exercises, period)
         if not exercise:
             return make_response(jsonify({"message": "Collection exercise not found"}), 404)
         full_exercise = collection_exercise_controller.get_collection_exercise_by_id(exercise['id'])
 
+        # Retrieve the events and format correctly
+        exercise_events = collection_exercise_controller.get_collection_exercise_events(exercise['id'])
+        formatted_events = {}
+        for event in exercise_events:
+            formatted_event_datetime = convert_event_to_new_format(event)
```

Comment:
Feels like we should be returning an ISO 8601 date here and leaving it to the client to decide the format the date should be shown in as it's more of a presentation thing

Reply:
Yeah that does feel better.
I've generally been trying to do the manipulation on the data that the UI is using in backstage but in this case it's probably better in the UI

Reply:
Done